docs/user-guide/diffing.md

```diff
@@ -116,6 +116,23 @@ data:
 
 By default `status` field is ignored during diffing for `CustomResourceDefinition` resource. The behavior can be extended to all resources using `all` value or disabled using `none`.
 
+### Ignoring RBAC changes made by AggregateRoles
+
+>v1.5
```

Comment:
Please remove this version indicator. I know they are still spread through the docs, but we're trying to get rid of those ever since we introduced versioned documentation :)